docs/source/api/plugin/schema-reporting.mdx

```diff
@@ -3,14 +3,15 @@ title: "API Reference: Schema reporting plugin"
 api_reference: true
 ---
 
+> Note:
+> **Schema reporting does not currently support graphs that use Apollo Federation.** If you would like to publish your monolith schema in your CI/CD pipeline, or if you are using Federation, you can use the [Rover CLI to publish your schema or subgraph changes](https://www.apollographql.com/docs/graphos/schema/cli-registration).
```

Comment:
Unless we know that this will change imminently, we should avoid the use of currently. (See https://developers.google.com/style/timeless-documentation)

```suggestion
> **The schema reporting plugin does not support graphs that use Apollo Federation.** If you are using Federation, or if you want to publish your monolith schema in your CI/CD pipeline, you can use the [Rover CLI to publish your schema or subgraph changes](/graphos/schema/cli-registration).
```